README.md

```diff
@@ -15,6 +15,18 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+## Answer 
+
+We will use _V_ as the vertices in this code. In this code, there is two loops on dealing with the rows in the matrix and the other dealing with the columns in the matrix too. With those loops, it would be _V_ x _V_. This would come out to be $V^2$ and the runtime analysis should be $\Theta(|V|^2)$.
+
+## Sources 
+
+I looked at Caden's repo for an idea of how I would write my pseudocode. This is all my comments in my code. I also wanted to challenge myself to write it in one function. 
```

Comment:
Not sure if I understand -- are you saying that the code came from somewhere else, but the comments are yours?

Reply:
I clarified better in the readme file.